src/flask/app.py

```diff
@@ -335,79 +170,33 @@ class Flask(_PackageBoundObject):
         }
     )
 
-    #: The rule object to use for URL rules created.  This is used by
-    #: :meth:`add_url_rule`.  Defaults to :class:`werkzeug.routing.Rule`.
-    #:
-    #: .. versionadded:: 0.7
-    url_rule_class = Rule
+    def __init__(
+        self,
+        import_name,
+        template_folder,
+        root_path=None,
+        static_url_path=None,
+        static_folder="static",
+        instance_relative_config=False,
```

Comment:
Blueprints don't have instance config or instance folders, so these shouldn't be in the skeleton.